rainier-core/src/main/scala/com/stripe/rainier/sampler/LeapFrog.scala

```diff
@@ -127,9 +132,8 @@ private[sampler] case class LeapFrog(density: DensityFunction) {
     * @param l0 the initial number of steps
     * @param stepSize the current value of the leapfrog step size
     */
-  private def longestBatchStep(params: Array[Double],
-                               l0: Int,
-                               stepSize: Double)(implicit rng: RNG): Int = {
+  def longestBatchStep(params: Array[Double], l0: Int, stepSize: Double)(
```

Comment:
This is now public and returns the acceptance prob so that it can be used, if desired, during dual averaging.